multinet/uploaders/swagger/newick.yaml

```diff
@@ -0,0 +1,28 @@
+Upload a Newick file to a collection of tables
+---
+consumes:
+  - text/plain
+
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: data
+    in: body
+    description: Raw Newick text
+    schema:
+      type: string
+      example: ((raccoon:19.19959,bear:6.80041):0.84600,((sea_lion:11.99700, seal:12.00300):7.52973,((monkey:100.85930,cat:47.14069):20.59201, weasel:18.87953):2.09460):3.87382,dog:25.46154);
+
+responses:
+  200:
+    description: Newick data uploaded to tables
+    schema:
+      type: object
+      properties:
+        edgecount:
+          type: integer
+        nodecount:
+          type: integer
+
```

Comment:
Can return 400 if the validation fails

Reply:
b072389ce9c73639630888fffdec8eb4d4bd0f65